src/components/ScholarshipDetailCardCell.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles, Grid, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    color: 'rgba(0, 0, 0, 0.6)',
+    padding: `${theme.spacing(1.5)}px 0`,
```

Comment:
What is this padding for? For creating the spacing between items? If so, I recommend adding padding for the dividers in between instead in ScholarshipDetailCard.jsx.